devops/incident-management.md

```diff
@@ -0,0 +1,91 @@
+- **Findings,**, e.g., calling out a piece of information that might be relevant to troubleshooting, such as a log message or the state of a system
+
+Encourage other folks who are engaged in incident response to share their updates as well.
+
+### Send hourly email updates
+
+Send updates to stakeholders hourly or as soon as you have any new information to share (whichever comes first). Delegate the hourly updating responsibility as needed.
+
+### Follow-up: Write a postmortem
+
+Every **SEV1** and **SEV2** incident must have a postmortem written within a week of its conclusion. The person who was most closely involved in the incident creates the postmortem but will usually include additional input from anyone else who aided in the response.
+
+You can create a postmortem from an incident by clicking the "+ New Postmortem Report" button on the incident page. The form will walk you through the required information. When the postmortem includes enough information about the incident, create a PDF and post it to the #[department]-incident-response channel in Slack for review by the rest of your team or department.
+
+For more details on PagerDuty's postmortem feature, consult the [PagerDuty Knowledge Base](https://support.pagerduty.com/docs/postmortems).
+
+Once the postmortem document is complete:
+
+- Save it as a PDF and add it to the archive folder
+- Create Asana tasks for any pending work raised in the "Prevention" section:
+  - Add each task to the Incident After-Actions project for tracking
+  - Assign the task to (or tag in) the relevant folks who will triage the task and ultimately complete it
+  Request an approximate due date for the task to determine whether to file the request as "Short Term" or "Long Term" section of the Incident After-Actions project.
+
+## For Low-Urgency Incidents (SEV3/SEV4/SEV5)
+
+### Tailor the response to the impact
+
+Low-urgency incidents have a lower expectation for how to respond and how quickly the incident gets resolved. A low-urgency incident is not a "drop everything" situation. However, there are still some actions to take at your earliest convenience. In PagerDuty, an incident categorized as low-urgency will alert using different notification rules and won't escalate if left unacknowledged.
+
+It's still important to verify the issue and notify stakeholders, but any response beyond that can wait until business hours. Product stakeholders may choose to be more proactive in their response, but they are not required to do so.
+
+Individual runbooks should contain more information about the expectations for how to respond to a low-urgency incident.
+
+### Update Slack with the status
+
+Use the #[department]-incident-response Slack channel to share all updates regarding the incident.
+
+### Send email updates as needed
+
+Send email updates to stakeholders as new information is available.
+
+### Follow-up: Track what's needed
```

Comment:
This could use some text underneath the header:

```suggestion
### Follow-up: Track what's needed

Create Asana tasks for any pending work that will address the cause of the incident.
```